### DISCUSSION TITLE
After cleaning the filter by date the table still shows the old results.

### DISCUSSION THREAD

**Discussion by M4x28** [Power-Components/livewire-powergrid#643] [BUG]
I was testing the datepicker but I noticed that after cleaning the filter by date, the table still shows the old results.
After filtering
![image](https://user-images.githubusercontent.com/64036726/193330138-4b34695c-bb88-4cab-bb39-3ced55b5ef1b.png)
After cleaning the filter
![image](https://user-images.githubusercontent.com/64036726/193330471-d7974c17-6b58-4289-92ac-62ab0dc37df1.png)

**Comment by luanfreitasdev** [comment]
I cannot reproduce this problem.

**Comment by Naagarjunaa** [comment]
Hi Powergrid team,
i am also facing this issue... it would be helpful if we get solution for this date picker issue
please find the video for reproducing issue.

Thanks for great package
[datepicker issue.webm](https://user-images.githubusercontent.com/46925151/196690462-06ad620c-3ede-4f9d-8834-178945897246.webm)

**Comment by Naagarjunaa** [comment]
Any Update on this query?

**Reply by M4x28** [reply]
Not still
